lib/puppet_litmus/rake_tasks.rb

```diff
@@ -377,6 +377,9 @@ def run_local_command(command)
         # The host_name might be localhost or an IP or a DNS-resolvable node.
         prefix = 'litmus:acceptance:'
         tasks = Rake::Task.tasks.select { |task| task.name =~ %r{^#{prefix}.+:\d+$} }
+        if tasks.empty?
+          tasks = Rake::Task.tasks.select { |task| task.name =~ %r{delivery.puppetlabs.net$$}; }
```

Comment:
Ahhhhhhhhhhh I understand now why we were finding them locally and failing via vmpooler! Whoops.